src/views/mfa-verify/PushForm.js

```diff
@@ -110,7 +130,9 @@ define(['okta', 'util/CookieUtil', 'util/Util'], function (Okta, CookieUtil, Uti
         });
         this.trigger('save', this.model);
         warningTimeout = Util.callAfterTimeout(_.bind(function () {
-          this.showWarning(Okta.loc('oktaverify.warning', 'login'));
+          if (!this.options.appState.get('isWaitingForNumberChallenge')) {
```

Comment:
A UX question - do we need a similar warning for number challenge?

Reply:
discussed with Steven and decided to hold of on any custom warning messages for number challenge as current text in the UI is descriptive enough.